### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by the-joksim**
* add update build instructions (covering Windows and MacOS as well)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,10 +40,6 @@ MINGW64 terminal:
 pacman -Syu --needed base-devel git mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl
 ```
 
-Once done, add the MINGW64 bin folder to the system PATH variable. If you've
-installed MSYS2 using the default installation settings, the bin folder path
-should be `C:\msys64\mingw64\bin`.
-
```
Comment (antonio2368) [inline README.md]: Do you still need this for the static linking?

Comment (the-joksim) [reply README.md]: Removed.

Comment (antonio2368) [reply README.md]: But the build won't work if the user doesn't add the path, so this needs to be returned?

Comment (the-joksim) [reply README.md]: No, it should work because the mingw console adds mingw64/bin to the `PATH`.

**Review by antonio2368 — APPROVED** [review]

**Pull request merged**